Not all Tests Passing on Fork

I just forked this repo and ran all the test off Master. I get a bunch of error. Am I missing something?

```bash
 jt ~/projects/node-mysql2 (master) $npm test

> mysql2@0.15.8 test /home/jt/projects/node-mysql2
> node ./test/run.js

[0:00:00 0 1/52 1.9% node test/integration/connection/test-binary-charset-string                                                                                
[0:00:00 0 2/52 3.8% node test/integration/connection/test-binary-charset-string                                                                                
[0:00:00 0 2/52 3.8% node test/integration/connection/test-binary-multiple-resul                                                                                
[0:00:00 0 3/52 5.8% node test/integration/connection/test-binary-multiple-resul                                                                                
[0:00:00 0 3/52 5.8% node test/integration/connection/test-binary-notnull-nulls.                                                                                
[0:00:00 0 4/52 7.7% node test/integration/connection/test-binary-notnull-nulls.                                                                                
[0:00:00 0 5/52 9.6% node test/integration/connection/test-change-user.js]   
  
  /home/jt/projects/node-mysql2/test/integration/connection/test-change-user.js:15
    if (err) throw err;
                   ^
  Error: Connection lost: The server closed the connection.
      at Socket.<anonymous> (/home/jt/projects/node-mysql2/lib/connection.js:107:15)
      at Socket.emit (events.js:129:20)
      at _stream_readable.js:908:16
      at process._tickCallback (node.js:355:11)
  
[0:00:00 1 5/52 11.5% node test/integration/connection/test-connect-sha1.js]
  
  events.js:85
        throw er; // Unhandled 'error' event
              ^
  Error: listen EADDRINUSE
      at exports._errnoException (util.js:746:11)
      at Server._listen2 (net.js:1156:14)
      at listen (net.js:1182:10)
      at Server.listen (net.js:1267:5)
      at Server.listen (/home/jt/projects/node-mysql2/lib/server.js:25:23)
      at Object.<anonymous> (/home/jt/projects/node-mysql2/test/integration/connection/test-connect-sha1.js:17:8)
      at Module._compile (module.js:460:26)
      at Object.Module._extensions..js (module.js:478:10)
      at Module.load (module.js:355:32)
      at Function.Module._load (module.js:310:12)
  
[0:00:00 2 5/52 13.5% node test/integration/connection/test-custom-date-parameter.js]
  
  [ { t: 650048400 } ] [ { catalog: 'def',
      schema: '',
      name: 't',
      orgName: '',
      table: '',
      orgTable: '',
      characterSet: 63,
      columnLength: 11,
      columnType: 8,
      flags: 128,
      decimals: 0 } ]
  
  assert.js:86
    throw new assert.AssertionError({
          ^
  AssertionError: [ { t: 650048400 } ] deepEqual [ { t: 650073600 } ]
      at process.<anonymous> (/home/jt/projects/node-mysql2/test/integration/connection/test-custom-date-parameter.js:24:10)
      at process.emit (events.js:107:17)
  
                                                                                
[0:00:00 3 5/52 15.4% node test/integration/connection/test-custom-date-paramete                                                                                
[0:00:01 3 5/52 15.4% node test/integration/connection/test-date-parameter.js] 
  
  [ { t: 631123200 } ] [ { catalog: 'def',
      schema: '',
      name: 't',
      orgName: '',
      table: '',
      orgTable: '',
      characterSet: 63,
      columnLength: 11,
      columnType: 8,
      flags: 128,
      decimals: 0 } ]
  
  assert.js:86
    throw new assert.AssertionError({
          ^
  AssertionError: [ { t: 631123200 } ] deepEqual [ { t: 631152000 } ]
      at process.<anonymous> (/home/jt/projects/node-mysql2/test/integration/connection/test-date-parameter.js:16:10)
      at process.emit (events.js:107:17)
  
[0:00:01 4 5/52 17.3% node test/integration/connection/test-datetime.js]       
  
  [ { d1: '1990-01-01',
      d2: '2000-03-03 00:15:11',
      d3: '2010-12-10 14:12:09' } ]
  [ { d1: '1990-01-01',
      d2: '2000-03-03 00:15:11',
      d3: '2010-12-10 14:12:09' } ]
  
  assert.js:86
    throw new assert.AssertionError({
          ^
  AssertionError: 8 == 0
      at process.<anonymous> (/home/jt/projects/node-mysql2/test/integration/connection/test-datetime.js:60:10)
      at process.emit (events.js:107:17)
  
[0:00:01 5 5/52 19.2% node test/integration/connection/test-disconnects.js]
  
  events.js:85
        throw er; // Unhandled 'error' event
              ^
  Error: listen EADDRINUSE
      at exports._errnoException (util.js:746:11)
      at Server._listen2 (net.js:1156:14)
      at listen (net.js:1182:10)
      at Server.listen (net.js:1267:5)
      at Server.listen (/home/jt/projects/node-mysql2/lib/server.js:25:23)
      at Object.module.exports.createServer (/home/jt/projects/node-mysql2/test/common.js:99:10)
      at Object.<anonymous> (/home/jt/projects/node-mysql2/test/integration/connection/test-disconnects.js:42:17)
      at Module._compile (module.js:460:26)
      at Object.Module._extensions..js (module.js:478:10)
      at Module.load (module.js:355:32)
  
[0:00:01 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:02 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:03 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:04 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:05 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:06 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:07 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:08 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:09 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:10 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:11 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:12 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:13 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:14 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:15 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:16 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:17 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:18 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:19 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:20 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:21 6 8/52 26.9% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:21 6 9/52 28.8% node test/integration/connection/test-execute-and-unprepar                                                                                
[0:00:21 6 10/52 30.8% node test/integration/connection/test-execute-newdecimal.                                                                                
[0:00:22 6 10/52 30.8% node test/integration/connection/test-execute-newdecimal.                                                                                
[0:00:22 6 11/52 32.7% node test/integration/connection/test-execute-newdecimal.                                                                                
[0:00:22 6 11/52 32.7% node test/integration/connection/test-execute-nocolumndef                                                                                
[0:00:22 6 12/52 34.6% node test/integration/connection/test-execute-nocolumndef                                                                                
[0:00:22 6 12/52 34.6% node test/integration/connection/test-execute-null-bitmap                                                                                
[0:00:22 6 13/52 36.5% node test/integration/connection/test-execute-null-bitmap                                                                                
[0:00:22 6 14/52 38.5% node test/integration/connection/test-execute-type-castin                                                                                
[0:00:22 6 15/52 40.4% node test/integration/connection/test-execute-type-castin                                                                                
[0:00:22 6 15/52 40.4% node test/integration/connection/test-insert-bigint-big-n                                                                                
[0:00:22 6 16/52 42.3% node test/integration/connection/test-insert-bigint-big-n                                                                                
[0:00:23 6 18/52 46.2% node test/integration/connection/test-invalid-date-result                                                                                
[0:00:23 6 19/52 48.1% node test/integration/connection/test-invalid-date-result                                                                                
[0:00:23 6 20/52 50.0% node test/integration/connection/test-multiple-results.js                                                                                
[0:00:23 6 21/52 51.9% node test/integration/connection/test-multiple-results.js                                                                                
[0:00:23 6 21/52 51.9% node test/integration/connection/test-named-paceholders.j                                                                                
[0:00:23 6 22/52 53.8% node test/integration/connection/test-named-paceholders.j                                                                                
[0:00:23 6 22/52 53.8% node test/integration/connection/test-nested-tables-query                                                                                
[0:00:23 6 23/52 55.8% node test/integration/connection/test-nested-tables-query                                                                                
[0:00:24 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:25 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:26 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:27 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:28 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:29 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:30 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:31 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:32 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:33 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:34 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:35 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:36 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:37 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:38 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:39 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:40 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:41 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:42 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:43 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:44 6 26/52 61.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:44 6 27/52 63.5% node test/integration/connection/test-prepare-and-close.j                                                                                
[0:00:44 6 28/52 65.4% node test/integration/connection/test-prepare-then-execut                                                                                
[0:00:44 6 29/52 67.3% node test/integration/connection/test-prepare-then-execut                                                                                
[0:00:44 6 29/52 67.3% node test/integration/connection/test-protocol-errors.js]
  
  events.js:85
        throw er; // Unhandled 'error' event
              ^
  Error: listen EADDRINUSE
      at exports._errnoException (util.js:746:11)
      at Server._listen2 (net.js:1156:14)
      at listen (net.js:1182:10)
      at Server.listen (net.js:1267:5)
      at Server.listen (/home/jt/projects/node-mysql2/lib/server.js:25:23)
      at Object.module.exports.createServer (/home/jt/projects/node-mysql2/test/common.js:99:10)
      at Object.<anonymous> (/home/jt/projects/node-mysql2/test/integration/connection/test-protocol-errors.js:3:27)
      at Module._compile (module.js:460:26)
      at Object.Module._extensions..js (module.js:478:10)
      at Module.load (module.js:355:32)
  
                                                                                                                                                                
[0:00:44 7 29/52 69.2% node test/integration/connection/test-quit.js]
  
  events.js:85
        throw er; // Unhandled 'error' event
              ^
  Error: listen EADDRINUSE
      at exports._errnoException (util.js:746:11)
      at Server._listen2 (net.js:1156:14)
      at listen (net.js:1182:10)
      at Server.listen (net.js:1267:5)
      at Server.listen (/home/jt/projects/node-mysql2/lib/server.js:25:23)
      at Object.module.exports.createServer (/home/jt/projects/node-mysql2/test/common.js:99:10)
      at Object.<anonymous> (/home/jt/projects/node-mysql2/test/integration/connection/test-quit.js:5:27)
      at Module._compile (module.js:460:26)
      at Object.Module._extensions..js (module.js:478:10)
      at Module.load (module.js:355:32)
  
[0:00:44 8 30/52 73.1% node test/integration/connection/test-select-empty-string                                                                                
[0:00:45 8 31/52 75.0% node test/integration/connection/test-select-empty-string                                                                                                                                                                                                                                                                                                                                
[0:00:45 8 38/52 88.5% node test/integration/connection/test-transaction-commit.                                                                                
[0:00:46 8 38/52 88.5% node test/integration/connection/test-transaction-commit.                                                                                
[0:00:46 8 39/52 90.4% node test/integration/connection/test-transaction-commit.                                                                                
[0:00:46 8 39/52 90.4% node test/integration/connection/test-transaction-rollbac                                                                                
[0:00:46 8 40/52 92.3% node test/integration/connection/test-transaction-rollbac                                                                                
[0:00:46 8 40/52 92.3% node test/integration/connection/test-type-cast-null-fiel                                                                                
[0:00:46 8 41/52 94.2% node test/integration/connection/test-type-cast-null-fiel                                                                                
[0:00:46 8 44/52 100.0% node test/unit/protocol/test-SqlString.js]            
npm ERR! Test failed.  See above for more details.
```

--- a/test/common.js
+++ b/test/common.js
@@ -76,7 +76,7 @@ module.exports.createConnection = function (args, callback) {
     typeCast: args && args.typeCast
   };
 
-  //console.log('cc params', params);
+  // console.log('cc params', params);
   var conn = driver.createConnection(params);
 
   /*

--- a/test/run.js
+++ b/test/run.js
@@ -7,4 +7,8 @@ var options = {
 if (process.env.FILTER) {
   options.include = new RegExp(process.env.FILTER + '.*\\.js$');
 }
+
+// set timezone to UTC
+process.env.TZ = 'UTC';
+
 require('urun')(__dirname, options);

